src/retrace/retrace.py

```diff
@@ -410,9 +326,9 @@ def guess_release(package, plugins):
 
 def get_supported_releases():
     result = []
-    files = os.listdir(CONFIG["RepoDir"])
+    files = os.listdir(conf.CONFIG["RepoDir"])
```

Comment:
You have overloaded `__getitem__()` of the conf object to load configuration if not loaded yet but here you call `__getitem__()` of the CONFIG member. Is the correct statement `conf["RepoDir"]`?